assets/javascripts/discourse/components/chat-user-displayname.js

```diff
@@ -0,0 +1,33 @@
+import Component from "@ember/component";
+import { computed } from "@ember/object";
+import { formatUsername } from "discourse/lib/utilities";
+
+export default class ChatUserDisplayname extends Component {
+  tagName = "";
+  user = null;
+
+  @computed
+  get shouldPrioritizeNameInUx() {
+    return !this.siteSettings.prioritize_username_in_ux;
```

Comment:
Oh no.. I was going to nit about the word `ux` then I realised this site setting isn’t chat specific.